tests/unit/worker/test_submission_worker.py

```diff
@@ -67,6 +90,72 @@ def setUp(self):
             ),
         )
 
+        self.participant_team = ParticipantTeam.objects.create(
+            team_name="Some Participant Team", created_by=self.user
+        )
+
+        self.challenge_phase = ChallengePhase.objects.create(
+            name="Challenge Phase",
+            description="Description for Challenge Phase",
+            leaderboard_public=False,
+            is_public=True,
+            start_date=timezone.now() - timedelta(days=2),
+            end_date=timezone.now() + timedelta(days=1),
+            challenge=self.challenge,
+            test_annotation=SimpleUploadedFile(
+                "test_sample_file.txt",
+                b"Dummy file content",
+                content_type="text/plain",
+            ),
+            max_submissions_per_day=100,
+            max_submissions_per_month=500,
+            max_submissions=1000,
+            codename="Phase Code Name",
+        )
+
+        self.submission = Submission.objects.create(
+            participant_team=self.participant_team,
+            challenge_phase=self.challenge_phase,
+            created_by=self.challenge_host_team.created_by,
+            status="submitted",
+            input_file=SimpleUploadedFile(
+                "test_sample_file.txt",
+                b"Dummy file content",
+                content_type="text/plain",
+            ),
+            method_name="Test Method",
+            method_description="Test Description",
+            project_url=self.testserver,
+            publication_url=self.testserver,
+            is_public=True,
+            is_flagged=True,
+        )
+
+    def helper_return_submission_input_file_path(self, submission_id, input_file):
+        """Helper Method: Takes `submission_id` and `input_file` as input and
+        returns corresponding path to submmitted input file"""
+
+        input_file_name = os.path.basename(input_file.name)
+        return join(self.SUBMISSION_DATA_DIR, "{input_file}").format(
+            submission_id=submission_id,
+            input_file=input_file_name,
+        )
+
+    def helper_return_phase_annotation_file_path(
+            self,
+            challenge_id,
+            phase_id,
+            annotation_file):
+        """Helper Method: Takes `challenge_id`, `phase_id` and `annotation_file`
+        as input and returns corresponding path to annotation file"""
+
+        annotation_file_name = os.path.basename(annotation_file.name)
+        return join(self.PHASE_DATA_DIR, "{annotation_file}").format(
+            challenge_id=challenge_id,
+            phase_id=phase_id,
+            annotation_file=annotation_file_name,
+        )
```

Comment:
Why is this method not being used anywhere?

Reply:
Yes. While I was importing the class variables from `submission_worker.py`, I thought it would be good to import all of them since there are other missing tests as well. The other tests such as that for `extract_challenge_data` would use these additions. It is in the GCI tasks so it would be helpful then. Also, it had almost the same template as the other helper method, so I thought I could add it now itself.

Reply:
I have made these gists: https://gist.github.com/nikochiko/c259554d3b2d2ea5cbea3fdfa65576f1 and https://gist.github.com/nikochiko/de6265470affbe221b41e3f77d46c09c for future reference. Will remove the unused methods and class variables for now.